oss.gg/7_create_a_template.md

```
@@ -54,4 +54,6 @@ Your turn 👇
 
 » 12-October-2024 by Nazar Poshtarenko [Next.js Unkey Pay-as-you-Go Starter Kit](https://github.com/unrenamed/unkey-nextjs-pay-as-you-go)
 
+» 18-October-2024 by Teddy ASSIH [AdonisJS + Unkey RBAC route protection ](https://github.com/Ionfinisher/unkey-adonisjs-template)
```

Markdownlint

Spaces inside link text

(MD039, no-space-in-links)